### DISCUSSION TITLE
Frontend Javascript

### DISCUSSION THREAD

**Discussion by vngantk** [vlang/v#17760] [Questions and Answers]
I am new to V and find it very interesting. I have a question. Is it possible to develop frontend Javascript applications using V? I know that V can produce Javascript code, but there is very little information talking about it. For example, how can I access the DOM? How can I call other Javascript libraries?

**Comment by HackFoxnew** [comment]
> I am new to V and find it very interesting. I have a question. Is it possible to develop frontend Javascript applications using V? I know that V can produce Javascript code, but there is very little information talking about it. For example, how can I access the DOM? How can I call other Javascript libraries?
> 

You should first investigate what WebAsm and Emscripten are and read the documentation to save you headaches https://emscripten.org/docs/index.html

**Comment by enghitalo** [comment]
> I am new to V and find it very interesting. I have a question. Is it possible to develop frontend Javascript applications using V? I know that V can produce Javascript code, but there is very little information talking about it. For example, how can I access the DOM? How can I call other Javascript libraries?

You can see https://github.com/vlang/v/tree/master/examples/js_dom_draw_bechmark_chart
